mbuild/compound.py

```diff
@@ -1237,16 +1237,7 @@ def _visualize_py3dmol(self, show_ports=False, color_scheme={}):
                                 'color':'grey'},
                         'sphere': {'scale': 0.3,
                                     'colorscheme':modified_color_scheme}})
-        if show_ports:
```

Comment:
Does this argument still work? Is this change related to the double-viewing?

Reply:
Yeah `show_ports` would still work. The way these viz routines work is by dumping compounds to a temporary mol2 file (with or without ports depending on the arg), and then visualizing that mol2 file. The delete `if show_ports` block was redundantly adding port particles when they were already visualized via the mol2-with-ports-included